src/containers/AlbumPhotos.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+
+import { fetchAlbumPhotosStatsById } from '../actions/albums'
+
+import PhotoBoard from './PhotoBoard'
+import Topbar from '../components/Topbar'
+
+export class AlbumPhotos extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      photosAreDirty: false
+    }
+  }
+
+  componentWillUpdate (nextProps, nextState) {
+    this.state.photosAreDirty = nextProps.photos &&
```

Comment:
You must not alter the state directly, but use `setState` instead. BUT I think it won't really work here because setting state in `componentWillUpdate` will trigger another rerender. You may want to find another way to do this.